### PR TITLE
This Pull request aims to introduce the SEP15

### PR DESCRIPTION
**Pull request by sagiss**
Title: Moving Sardana to Github
Abstract: Move Sardana project from its current hosting in SourceForge to 
 GitHub. The move affects the code repository, the ticket tracker and the 
 wiki pages. It also proposes to change the contribution and the SEP 
 workflow to make use of the Pull Request feature.

### CODE REVIEW

**Comment by reszelaz** [comment]
Hi all,

Finally the CANDIDATE implementation of the SEP15 (migration of Sardana to
GitHub) is ready to be reviewed.

Please review the proposed changes in:

https://github.com/sardana-org/sardana/pull/1

... and discuss in that same Pull Request if you have any comment

Also, a representative from each institute should cast a vote regarding
whether we should accept the SEP15 (please do it ASAP by commenting on the
Pull Request itself).

Note that ACCEPTING SEP15, automatically moves SEP0, SEP1, SEP7 and SEP10 to OBSOLETE
state.

Cheers,

Marc & Zibi

PD: please note that the "service-agnostic URL for the SEPs" mentioned in the
SEP15 will only work once we actually merge the Pull Request (i.e., do not
worry about broken links until then)

**Comment by LJBD** [comment]
Thumbs up from Solaris!

**Comment by teresanunez** [comment]
Also from DESY

**Comment by amilan** [comment]
👍  from MAX IV

**Comment by NexeyaSGara** [comment]
From Nexeya :)

File: doc/source/sep/index.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  [SEP11][]    | ACCEPTED     | Direct load of .ui files
+  [SEP12][]    | CANDIDATE     | Use python Enum instead of taurus Enumeration
+  [SEP13][]    | REJECTED (moved to [TEP13][])     | Unified plugins support in Taurus & Sardana
+  [SEP14][]    | DRAFT     | MSENV taurus schema
+  [SEP15][]   | CANDIDATE     | Moving Sardana to Github
+
+
+
+
+[SEP0]: sep/SEP0.md
```
Comment (cpascual) [inline doc/source/sep/index.md]: I think these links will be broken. I recomend using links such as:
http://www.taurus-scada.org/tep/?TEP13.md

Comment (reszelaz) [inline doc/source/sep/index.md]: Changed, Thanks!

**Comment by reszelaz** [comment]
Thanks to all for voting! And to @sagiss for the great work!
I have merged it locally after first rebasing on top of develop and pushed it.

**Pull request closed without merging**